package.json

```diff
@@ -37,15 +38,16 @@
     "test:all": "mocha node-tests && ember try:each"
   },
   "dependencies": {
-    "@babel/plugin-transform-modules-amd": "^7.0.0-beta.55",
-    "@babel/preset-env": "^7.0.0-beta.55",
+    "@babel/core": "^7.0.0-beta.56",
```

Comment:
Seems slightly odd to force the `@babel/core` dep here. Was it to prevent peer dep warnings?

Reply:
Nope, it was actually because I was getting a `Cannot find module: '@babel/core'`. I'd assumed it was because some last transitive dep had finally shifted it from a dep to a peer dep, but looking at the tree now I think it's more likely that my `node_modules` was just in a weird place because yarn isn't super good at `git` dependencies...

The peerDep warnings _are_ super annoying, but weren't the motivator for this. Happy to revert now that I know it's not necessary.